desc/compute/_basis_vectors.py

```diff
@@ -1385,11 +1385,11 @@ def _e_sup_zeta_zz(params, transforms, profiles, data, **kwargs):
 
 
 @register_compute_fun(
-    name="e_phi",
-    label="\\mathbf{e}_{\\phi}",
+    name="e_phi|r,t",
+    label="\\mathbf{e}_{\\phi} |_{\\rho, \\theta}",
     units="m",
     units_long="meters",
-    description="Covariant cylindrical toroidal basis vector",
+    description="Covariant toroidal basis vector in (ρ,θ,Φ) coordinates",
```

Comment:
can you maybe make it clear in the description that it is the partial derivative of the position vector evaluated at constant rho, theta?